modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/thunks/fieldBlur.es.js

```diff
@@ -26,7 +26,7 @@ export default function fieldBlur({
 		dispatch({payload: properties, type: EVENT_TYPES.FIELD.BLUR});
 
 		Liferay.fire('ddmFieldBlur', {
-			fieldName: fieldInstance.fieldName,
+			fieldName: fieldInstance.label,
```

Comment:
@fcodias, if there is a form with 2 fields with the same label, won't it affect your side?

cc @interaminense

Reply:
That's a good question

Reply:
Just to give some context, for repeatable fields the label will be the same for each repeatable field. :wink:

Reply:
Thanks for the review @natocesarrego!
Do you know the behavior for this case in 7.0~7.2?
I mean if it's already duplicated I think it's ok to forward this fix and I'll raise up this issue with PMs and Designers, but, if it's kind different we should keep the same behavior..
If you guys don't know I can ask for QAs to validate this scenario in the previous versions..

Reply:
I just answered @fcodias through Slack and I'm waiting his reply before decide the next steps for this pull.

Reply:
Hey @natocesarrego, Just received an answer from the designer and we decided that we can go ahead using just the `label` since our interest is if the field was abandoned... Can we forward this one?

Thank you guys! @interaminense @natocesarrego

Reply:
Sure, @fcodias! :wink: